### ISSUE TITLE
Feature Request: Bonding Mode

### ISSUE THREAD

**Issue by grf1** [datarhei/gosrt#35]
Hello, 

Would you consider developing support for SRT bonding ? 

George

**Comment by ioppermann** [comment]
@grf1 Yes, it is planned to implement connection bonding. However, there's currently no schedule when it will be available. Pull requests and sponsorings are welcome.

**Comment by krsna1729** [comment]
Was trying out with an agent [here](https://github.com/krsna1729/gosrt/tree/bonding).
